src/amchart/CommonSerial.js

```diff
@@ -288,6 +291,7 @@
                 this._chart.valueAxes[0].logarithmic = false;
                 if (this.sortDates()) {
                     this.data(Utility.naturalSort(this.data(), "ascending", 0));
+                    this._dataUpdated--; // decrament the counter becuase its not a true data update
```

Comment:
perhaps i should just assign to a new variable ... just trying to use ness memory